### DISCUSSION TITLE
Welcome to essdive-uas Discussions!

### DISCUSSION THREAD

**Discussion by wavingtowaves** [ess-dive-community/essdive-uas#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by regnans** [comment]
thanks for getting this discussion forum going! We're hoping to get feedback on the draft reporting format at all levels. From the smallest detail, to big picture "what should the content of a UAS data package be" stuff. All feedback is welcome and will be carefully considered.
